develop/api_guc.rst

```diff
@@ -66,6 +66,23 @@ $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
 Sets the time to wait before checking for distributed deadlocks. In particular the time to wait will be this value multiplied by PostgreSQL's `deadlock_timeout <https://www.postgresql.org/docs/current/static/runtime-config-locks.html>`_ setting. The default value is ``2``. A value of ``-1`` disables distributed deadlock detection.
 
+.. _node_connection_timeout:
+
+citus.node_connection_timeout (integer)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The ``citus.node_connection_timeout`` GUC sets the maximum duration (in milliseconds) to wait while connecting to worker nodes.
```

Comment:
> `to wait while connecting to worker nodes.` 

Maybe (please improve the grammer/language): 


The `citus.node_connection_timeout` GUC sets the maximum duration (in milliseconds) for connection establishment. When this is exceeded before at least one connection per worker node is established, Citus errors out.


Note (feel free to ignore): For deprecated task-tracker executor, Citus errors out if not all the required connections within `citus.node_connection_timeout` is established